src/repositories/concrete/levelBlockchainRepository.ts

```diff
@@ -18,15 +18,35 @@ export class LevelBlockchainRepository implements IBlockchainRepository {
         throw new Error("Method not implemented.");
     }
 
+    public async getBlocksByPeriodAsync(beginTimestamp: number, endTimestamp: number): Promise<Block[]> {
+        throw new Error("Method not implemented.");
+    }
+
+    public async getBlocksByDateAsync(timestamp: number): Promise<Block[]> {
```

Comment:
Same here look at the other comment for the explanation (line 43)

I will stop commenting on these mistakes now as they are naturally repeated across the repository pattern. 😄